dist/ci/testenv-leap151/osc-hotpatch.diff

```diff
@@ -0,0 +1,11 @@
+--- /usr/lib/python3.6/site-packages/osc/core.py.bak    2019-05-02 18:14:01.088380882 +0000
++++ /usr/lib/python3.6/site-packages/osc/core.py        2019-05-02 18:14:19.204341426 +0000
+@@ -6318,7 +6318,7 @@
+         try:
+             comment = node.find('comment').text.encode(locale.getpreferredencoding(), 'replace')
+         except:
+-            comment = b'<no message>'
++            comment = '<no message>'
```

Comment:
btw: the patch is most likely not correct, but it helps pass the test suite and that's all that counts.